includes/reports/data/class-report-registry.php

```diff
@@ -119,7 +119,7 @@ public function add_report( $report_id, $attributes ) {
 		$attributes = array_merge( $defaults, $attributes );
 
 		try {
-			$this->validate_attributes( $attributes, $report_id );
+			$this->validate_attributes( $attributes, $report_id, array( 'filters' ) );
```

Comment:
Can you add a comment above that just says filters are excluded because we allow it to be empty?